rdmo/core/imports.py

```diff
@@ -134,15 +135,20 @@ def track_changes_on_element(element: dict,
 
     _initialize_track_changes_element_field(element, element_field)
 
+    lookup_model_field = None
     if original_value is None and original is not None:
         lookup_field = element_field if instance_field is None else instance_field
         original_value = getattr(original, lookup_field, '')
+        lookup_model_field = original._meta.get_field(lookup_field)
 
     if isinstance(new_value,str) and isinstance(original_value,int):
         # typecasting of original value to str, for comparison '0' == 0
         # specific edge-case, maybe generalize later
         original_value = str(original_value)
 
+    if isinstance(lookup_model_field, FloatField):
+        new_value = float(new_value)
```

Comment:
Should I catch a `ValueError` here @jochenklar ?
This value comes directly from the uploaded xml, for example for the field `Question.maximum` and 
when there is a typo from the user it might raise a `ValueError`.

Reply:
Other cases for validation `widget_type`, `value_type`.

Reply:
> Yes, you can set the field empty or propagate an error.

Ok, I set it empty (`None`) for now